terraform/main.tf

```diff
@@ -69,15 +69,30 @@ module "gh_oidc" {
   }
 }
 
+# 2024-04-18: separate from the others because this was the first one - if we
+# combined the two, this would delete and recreate the service account
 resource "google_service_account" "service_account" {
   account_id   = "rmi-beta-access"
-  display_name = "RMI Beta Access"
+  display_name = "rmi_beta_access"
+}
+
+# 2024-04-18: after creating a new SA you will have to also create a keypair
+# for the user.
+resource "google_service_account" "beta_access_service_accounts" {
```

Comment:
Are there any google resources that get named: `beta_access_service_accounts` or is this just the name of the terraform step/resource?

Reply:
This is the name of the resource *in Terraform* - Google sees the `account_id` as the actual ID of each service account.